experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/proxies.ts

```diff
@@ -497,3 +514,94 @@ export function createListProxy<TTypes extends AllowedTypes>(
 		},
 	});
 }
+
+// #region Create dispatch map for maps
+
+const mapStaticDispatchMap: PropertyDescriptorMap = {};
+
+[
+	/* eslint-disable @typescript-eslint/unbound-method */
+	Map.prototype.clear,
+	Map.prototype.delete,
+	Map.prototype.entries,
+	Map.prototype.forEach,
+	Map.prototype.get,
+	Map.prototype.has,
+	Map.prototype.keys,
+	Map.prototype.set,
+	Map.prototype.values,
+	/* eslint-enable @typescript-eslint/unbound-method */
+].forEach((fn) => {
+	listPrototypeProperties[fn.name] = { value: fn };
```

Comment:
Whoops.  Don't add them to list's prototype.  :-)

Reply:
Oof, that was definitely bound to the correct variable earlier. I don't know how I messed that up in the merge.

Okay, I was wondering if that might be the case. I'd discovered that get didn't work as expected, but I was hoping some of the other methods might "just work". If that's not the case, I'll go ahead and just clear this out altogether and leave placeholders in the function.